src/basicEventScheduler.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
please update the copyright info, (The above info is used by raven). Something like:

```
# Copyright 2020, Battelle Energy Alliance, LLC
# ALL RIGHTS RESERVED
```

Reply:
Fixed